pycombo/pyCombo.py

```diff
@@ -38,21 +41,29 @@ def execute(
     weight : str, default 'weight'
         Graph edges property to use as weights. If None, graph assumed to be unweighted.
         Ignored if graph is passed as string (path to the file).
-    max_communities : int, default -1
-        Maximum number of communities. If -1, assume to be infinite.
+    max_communities : int, default None
+        Maximum number of communities. If <= 0 or None, assume to be infinite.
     modularity_resolution : float, default 1.0
         Modularity resolution parameter.
     num_split_attempts : int, default 0
         Number of split attempts. If 0, autoadjust this number automatically.
     fixed_split_step : int, default 0
         Step number to apply predefined split. If 0, use only random splits,
         if >0 sets up the usage of 6 fixed type splits on every fixed_split_step.
+    start_separate : bool, default False
+        Indicates if Combo should start from assigning each node into its own separate community.
+        This could help to achieve higher modularity, but it makes execution much slower.
     treat_as_modularity : bool, default False
         Indicates if edge weights should be treated as modularity scores.
         If True, the algorithm solves clique partitioning problem over the given graph,
         treated as modularity graph (matrix).
         For example, this allows users to provide their own custom 'modularity' matrix.
         `modularity_resolution` is ignored in this case.
+    info_output_level : int, default 0
```

Comment:
I think sklearn uses term `verbose`
https://stackoverflow.com/questions/29995249/verbose-argument-in-scikit-learn

also, google showed this post - I wonder if long-term we could sync python log levels with the verbosity (probably not worth it, though)
https://medium.com/@kalebporter/logging-extending-python-with-c-or-c-fa746466b602

Reply:
Thanks, yes, we can use `verbose` instead.

As for logging, I agree that it is not worth it, since we for now we only want to see some immediate progress messages in console.